tools/common/StringUtils.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Text;
+
+namespace Xamarin.Utils {
+	public class StringUtils {
+
+		static char shellQuoteChar;
+		static char ShellQuoteChar {
```

Comment:
Sounds better / simpler to have this computed in the .cctor and avoid the call (if not inclined) and the check (twice in quote)